components/permissions/sources.gni

```diff
@@ -24,12 +28,16 @@ brave_components_permissions_sources = [
 brave_components_permissions_deps = [
   "//base",
   "//brave/components/brave_wallet/browser:permission_utils",
+  "//brave/components/constants:constants",
+  "//brave/components/google_sign_in_permission",
   "//brave/components/l10n/common",
   "//brave/components/resources:strings_grit",
   "//components/content_settings/core/browser",
   "//components/content_settings/core/common",
   "//components/keyed_service/core",
   "//components/pref_registry",
+  "//components/user_prefs:user_prefs",
+  "//services/network/public/cpp",
```

Comment:
after moving throttle to `google_sign_in_permission`, these dependencies should be removed.